spec/solver_spec.rb

```diff
@@ -0,0 +1,41 @@
+require_relative '../solver'
+
+describe Solver do
+  before(:each) do
+    @solver = Solver.new
+  end
+
+  describe '#factorial' do
+    it('should check of factorial of a number') do
+      expect(@solver.factorial(3)).to eq 6
+    end
+
+    it('should check of factorial of a number') do
+      expect(@solver.factorial(5)).to eq 120
+    end
+  end
+
+  describe '#fizzbuzz' do
+    it 'Should verify if the number is multiple of 3 and 5' do
+      expect(@solver.fizzbuzz(15)).to eq 'fizzbuzz'
+    end
+
+    it 'Should verify if the number is multiple of 3' do
+      expect(@solver.fizzbuzz(6)).to eq 'fizz'
+    end
+
+    it 'Should verify if the number is multiple of 5' do
+      expect(@solver.fizzbuzz(10)).to eq 'buzz'
+    end
```

Comment:
- [ ] I would recommend testing the case where the number does not belong to `fizz`, `buzz`, or `fizzbuzz`.

```
expect(@solver.fizzbuzz(7)).to eql('7')
```